libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -167,7 +167,8 @@ class MapboxNavigation(
         null,
         electronicHorizonOptions,
         null,
-        incidentsOptions
+        incidentsOptions,
+        null
```

Comment:
What NN version are `'da-common-runloop-callback-3-SNAPSHOT'` and `'da-common-scheduler-callback-5-SNAPSHOT'` artifacts based off? Latest `master`? Could you clarify @DmitryAk?

Reply:
Both artifacts are based on 3-days-old `master` (they were provided on March 2nd).
Today I'm going to provide them (both) updated on current `master`